editor.planx.uk/src/components/Header.tsx

```diff
@@ -414,6 +415,15 @@ const EditorToolbar: React.FC<{
     setOpen(!open);
   };
 
+  const isFlowSettingsVisible =
+    route.data.flow && !route.data.flow && canUserEditTeam(team.slug);
+
+  const isTeamSettingsVisible =
+    route.data.team && !route.data.flow && canUserEditTeam(team.slug);
+
+  const isGlobalSettingsVisible =
+    !route.data.flow && !team.slug && user?.isPlatformAdmin;
```

Comment:
A few related fixes / tidy ups to the change made in this PR , apologies for polluting things a little.